packages/design-system/src/components/utilities/visibility.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+export default {
+  title: 'Utilities/Visibility',
+  parameters: {
+    docs: {
+      source: {
+        type: 'dynamic',
+      },
+    },
+  },
+};
+
+// an example for responsive layout grid -- referenced in doc site
+export const visibility = () => (
+  <div>
+    {['sm', 'md', 'lg', 'xl'].map((breakpoint) => (
+      <div className="ds-u-padding--1 ds-u-border--1" key={breakpoint}>
+        <div className={`ds-u-${breakpoint}-visibility--hidden ds-u-color--muted`}>
+          Hidden on {breakpoint} screens and larger
+        </div>
+        <div className={`ds-u-${breakpoint}-visibility--visible`}>
+          Visible on {breakpoint} screens and larger
+        </div>
+      </div>
+    ))}
```

Comment:
This is not related to code you added, but what I don't get about this example is that in the`xs` tab the "visible on sm screens and larger" is visible.

Reply:
Does it maybe need to be hidden by default like `ds-u-visibility--hiden ds-u-${breakpoint}-visibility--visible`?

Reply:
Updated, because you're right, it doesn't really make sense as-is